charts/kargo/values.yaml

```diff
@@ -198,6 +198,12 @@ api:
     clientID:
     ## @param api.oidc.cliClientID The client ID for the OIDC client used by CLI (optional). Needed by some OIDC providers (such as Dex) that require a separate Client ID for web app login vs. CLI login (`http://localhost`). If Dex is enabled, this value will be ignored and cli client ID will be automatically configured. If Dex is not enabled, and a different client app is configured for localhost CLI login, this should be the client ID configured in the IdP.
     cliClientID:
+    ## @param api.oidc.scopes The scopes to send to the OIDC provider. If Dex is enabled, this value will be ignored and the scopes should be in the dex connectors configuration. If Dex is not enabled, this should be set to the scopes you wish to be provided to Kargo by your identity provider.
+    scopes: 
+      - openid
+      - profile
+      - email
+      - groups
```

Comment:
A few things on this bit...

First, nicely documented!

I might prefer that we make this `additionalScopes` with a default value of `[]` and keep the list that you have here hard-coded. I believe we're going to want these four in nearly all cases, so hard-coding these and only requiring users to list _additional_ ones limits the opportunity for mis-configuration.

The other thing:

> If Dex is enabled, this value will be ignored and the scopes should be in the dex connectors configuration. If Dex is not enabled, this should be set to the scopes you wish to be provided to Kargo by your identity provider.

idk if that sounds right.

You're 100% correct that the Dex connector config should specify desired scopes for the upstream IDP.

But this list is for Kargo to tell _clients_ of the IDP what scopes they ought to ask for. (CLI and UI talk _directly_ to the IDP without Kargo as a middle-man.)

In the case where Dex _is_ the IDP the client will talk to, I think this list is still relevant.